doc/nrf/ug_unity_testing.rst

```diff
@@ -25,8 +25,8 @@ To run the unit test, enable the Unity module (:kconfig:option:`CONFIG_UNITY`) a
 The module under test and all dependencies are enabled and compiled into the binary, even the mocked modules.
 
 The linker replaces all calls to the mocked API with a mock implementation using the wrapping feature.
-For example, :code:`-Wl,--wrap=foo` replaces every call to :code:`foo()` with a call to :code:`__wrap_foo()`.
-Because of that, all mock functions are prefixed with :code:`__wrap` (for example, :code:`__wrap_foo_Expect()` instead of :code:`foo_Expect()`).
+For example, :code:`-Wl,--wrap=foo` replaces every call to :code:`foo()` with a call to :code:`__wrap_cmock_foo()`.
+Because of that, all mock functions are prefixed with :code:`__wrap` (for example, :code:`__wrap_cmock_foo_Expect()` instead of :code:`foo_Expect()`).
```

Comment:
```suggestion
Because of that, all mock functions are prefixed with :code:`__wrap_cmock` (for example, :code:`__wrap_cmock_foo_Expect()` instead of :code:`foo_Expect()`).
```

Reply:
thanks for spotting